pingu/corpus/assets/label.py

```diff
@@ -13,13 +13,15 @@ class Label(object):
         start (float): Start of the label within the utterance in seconds. (default: 0)
         end (float): End of the label within the utterance in seconds. (default: -1) (-1 defines
                      the end of the utterance)
+        label_list (LabelList): The label-list this label is belonging to.
     """
-    __slots__ = ['value', 'start', 'end']
+    __slots__ = ['value', 'start', 'end', 'label_list']
 
     def __init__(self, value, start=0, end=-1):
         self.value = value
         self.start = start
         self.end = end
+        self.label_list = None
```

Comment:
`label_list=None` as argument would make sense, no?

Reply:
The ``label_list`` attribute is set automatically, when the label is added to a label-list (in the append method). Therefore i think it is not useful.